code/game/objects/items/stacks/medical.dm

```diff
@@ -1,24 +1,34 @@
+#define REAGENT_AMOUNT_PER_ITEM 20 //The amount of reagents medical items contain, for both application and grinding purposes.
```

Comment:
this should be separated into
* REAGENT_AMOUNT_HEAL 20 // the amount of healing power
* REAGENT_AMOUNT_CONTAINED 40 // the amount of the medical items contain

Reply:
🤔 

Why on earth would I use two defines - defining how much is healed completely defeats the purpose of utilizing reagent code to determine how much is healed. The amount of reagents contained is **also** the healing efficacy because it transfers those reagents which then heals?